tools/jslint/check-all.js

```diff
@@ -60,7 +60,7 @@ var globals =
     + ' viewport process_loaded_for_unread'
     + ' have_scrolled_away_from_top'
     + ' maybe_scroll_to_selected recenter_pointer_on_display suppress_scroll_pointer_update'
-    + ' fast_forward_pointer recent_subjects unread_subjects'
+    + ' fast_forward_pointer recent_subjects unread_subjects recent_private_messages'
```

Comment:
Having thought about this more and recalled the history here, I think actually our goal is actually to purge all global variables from Zulip, and instead attach them as exports of the appropriate modules for the code.  So I think this should probably not be needed and we should instead call this `stream_list.recent_private_messages` (define it as `exports.recent_private_messages` within `stream_list.js`).